src/pre.js

```diff
@@ -90,21 +94,28 @@ var glpkPromise = new Promise(function (resolve) {
 					housekeeping(P);
 					return FS.readFile(name, { encoding: 'utf8' });
 				},
-				'solve': function (lp, msg_lev) {
-
+				'solve': function (lp, jsonSettings) {
+					
+					const settings = _.defaults(jsonSettings, {
+							msgLev: 1,
```

Comment:
can't we use GLP_MSG_ERR?

Reply:
No we can't here. it's just defined as a key o fapi obj and solve itself is another key.

Reply:
I see, this unfortunate... so maybe a comment here should be added to know from where the 1 comes from.